front/app/components/admin/PostManager/components/FilterSidebar/index.tsx

```diff
@@ -142,13 +142,14 @@ const FilterSidebar = ({
     phases: () => ({
       name: tabName('timelineTab', selectedPhase, 'phases'),
       key: 'phases',
-      content: (
-        <PhasesMenu
-          phases={phases}
-          selectedPhase={selectedPhase}
-          onChangePhaseFilter={onChangePhaseFilter}
-        />
-      ),
+      content:
+        phases && onChangePhaseFilter ? (
```

Comment:
It doesn't make sense to render a phases filter without these, so let's make our lives easier inside FilterSidebarPhases.